repose-aggregator/src/docs/asciidoc/filters/valkyrie-authorization.adoc

```diff
@@ -2,51 +2,224 @@
 
 include::../_includes/in-progress.adoc[]
 
+The Valkyrie Authorization filter performs three distinct functions:
+
+* **Per-Device Authorization**:
+Compares the request method with the user's device permissions to determine whether or not the requested action is allowed.
+* **Response Collection Resource Culling**:
+Values from the origin service's response will be removed based on the user's device permissions.
+Consequently, the user may only view items they have permission to view.
++
+[NOTE]
+====
+Currently, only JSON bodies are supported.
+If you would like to request XML support, please <<../contact-us.adoc#,contact us>>!
+====
+* **Account Permission to Role Translation**:
+Takes the user's account level permissions and translates their names into the standard `X-Roles` header allowing any of the standard means of RBAC to be used on the request.
+This includes translating the user's device permission for the device specified in the `X-Device-Id` header (when the request contains one).
+
+While the authorization function is always performed, the other functions may be enabled or disabled as desired.
```

Comment:
So we'll go ask for your info if we have the device id header, the resource matches a configured one, or if you've asked for roles translation.

https://github.com/rackerlabs/repose/blob/master/repose-aggregator/components/filters/valkyrie-authorization-filter/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala#L167-L168